dnf-behave-tests/dnf/upgrade.feature

```diff
@@ -207,7 +211,11 @@ Scenario Outline: Print correct number of available updates if update <type> is
    When I execute dnf with args "update <type>" 
    Then the exit code is 0
     And Transaction is empty
-    And stderr contains "No security updates needed, but 2 updates available"
```

Comment:
I believe we actually want this: https://github.com/rpm-software-management/dnf5/issues/360
I would keep the test disabled.

Reply:
the same as previous issue